### PR TITLE
Fix the run registry

### PR DESCRIPTION
**Pull request by plasorak**

### CODE REVIEW

**Comment by jcpunk** [comment]
Is this still viable for merge?

**Comment by plasorak** [comment]
Yes

**Review by jcpunk — APPROVED** [review]

**Comment by jcpunk** [comment]
I'd be tempted to run the python through the `black` autoformatter in a second commit for normalization.

File: runregistry-rest/rest.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__author__ = "Roland Sipos"
+_author__ = "Roland Sipos"
```
Comment (jcpunk) [inline runregistry-rest/rest.py]: Shouldn't this keep the `__`?

Comment (plasorak) [reply runregistry-rest/rest.py]: Yes, that's a mistake

File: runregistry-rest/rest.py [hunk 2]
```diff
@@ -17,111 +17,135 @@
...
-app.config['MAX_CONTENT_LENGTH'] = 32 * 1000 * 1000
-app.config['UPLOAD_EXTENSIONS'] = ['.gz', '.tgz']
-app.config['UPLOAD_PATH'] = 'uploads'
-app.config['CACHE_TYPE'] = 'redis' # easier to scale it and async disk writes provides DB dumps.
+app.config["MAX_CONTENT_LENGTH"] = 32 * 1000 * 1000
+app.config["UPLOAD_EXTENSIONS"] = [".gz", ".tgz"]
+app.config["UPLOAD_PATH"] = "uploads"
+app.config[
+    "CACHE_TYPE"
+] = "simple"  # easier to scale it and async disk writes provides DB dumps.
```
Comment (jcpunk) [inline runregistry-rest/rest.py]: I suspect the length of this comment is what caused black to do this odd layout.

Comment (plasorak) [reply runregistry-rest/rest.py]: fixed

**Comment by jcpunk** [comment]
I don't see anything in there that worries me, but I'm not familiar with what the code should be doing overall.

**Comment by plasorak** [comment]
I think @TiagoTAlves has tested and could undersign this?

**Comment by TiagoTAlves** [comment]
From the testing we did, it seems to work

**Comment by jcpunk** [comment]
Does that mean this is a "go for merge"?

**Pull request merged**